pytorch3dunet/bioimage-io/UNet3DArabidopsisOvules.model.yaml

```diff
@@ -0,0 +1,88 @@
+name: UNet3DArabidopsisOvules
+description: A 3d U-Net trained to predict the cell boundaries in confocal stacks of Arabidopsis ovules
+cite:
+  - text: "Wolny, Adrian et al. Accurate and Versatile 3D Segmentation of Plant Tissues at Cellular Resolution. BioRxiv 2020."
+    doi: https://doi.org/10.1101/2020.01.17.910562
+authors:
+  - Adrian Wolny;@bioimage-io
+documentation: ../README.md
+tags: [unet3d, pytorch, arabidopsis, ovuls, cell membrane, segmentation, plant tissue]
+
+format_version: 0.1.0
+language: python
+framework: pytorch
+
+# todo: import path with '..'  # add a root_dir?
+source: ..pytorch3dunet.model.UNet3D
```

Comment:
relative import paths with `..` should probably be avoided...
- Do we need a root_dir? with default `root_dir = "."`? < current favorite
- Is the repo dir `root_dir=repo_dir`? Do we then check  from "." and repo_dir?